apps/rotm/index.js

```diff
@@ -14,11 +18,19 @@ module.exports = {
       fields: [
         'more-info'
       ],
-      next: '/confirm'
+      next: '/image'
+    },
+    '/image': {
+      next: '/add-image',
+      behaviours: skipStep
+    },
+    '/add-image': {
+      next: '/check-your-report'
     },
-    '/confirm': {
+    '/check-your-report': {
       behaviours: ['complete'],
-      next: '/confirmation'
+      next: '/confirmation',
+      backLink: 'image'
```

Comment:
I'm not sure this is something we want. In the case where a user does upload an image then I imagine we want to take them to `/add-image`

Reply:
I think that's fair.  The issue is that if a user doesn't upload an image and they click the backlink, it puts them to the `/add-image` step.

Did you want me to add a behaviour for the different scenarios? I'll do it as a separate PR because this PR is holding up @JoeChapman work

Reply:
I suspect that's an artefact of your existing skip-step behaviour, which (incorrectly) adds `/add-image` to the list of steps which a user has completed.

If that is resolved, then the back link will automatically take the user to their last completed step, which would be `/image` or `/add-image` as appropriate.